### DISCUSSION TITLE
Scope definition for Client Credentials

### DISCUSSION THREAD

**Discussion by eric-murray** [camaraproject/IdentityAndConsentManagement#81] [Q&A]
Can somebody confirm that scopes for client credentials authorisation do **NOT** require a [purpose](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#applying-purpose-concept-in-the-authorization-request) to be included in the scope format, and so the scope consists **ONLY** of the `technicalParameter`, as defined in the API specification itself?

This may seem obvious, but it would be good to confirm and maybe update the documentation to clarify this.

**Comment by jpengar** [comment]
Duplicated. This topic is being discussed in https://github.com/camaraproject/IdentityAndConsentManagement/issues/92
